packages/code/src/highlight.d.ts

```diff
@@ -1,6 +1,7 @@
 interface TokenObject {
   kind: string;
   children: Array<string | TokenObject>;
+  sublanguage?: boolean;
```

Comment:
where do we use this property?

Reply:
We don't, but it can exist on the object. highlight.d.ts is an attempt to accurately type the highlight.js library (or at least, the parts we've come across).

Reply:
👍